.github/workflows/compat-android.yml

```diff
@@ -2,7 +2,7 @@ name: Compat / Android
 
 on:
   schedule:
-    - cron: "0 0 * * *" # Once a day at midnight
+    - cron: "0 0 * * 1" # Once a week at midnight on Monday
```

Comment:
I had actually assumed we would fall into GitHub's free tier because the project is open source. Thanks for actually monitoring the cost. 😳

Reply:
It does. However, I'd like to reduce energy use even if I'm not paying for it. 💚 🌍 🌳

Reply:
Oh, even better call then! 👍